gravity-forms/gw-create-coupon.php

```diff
@@ -56,6 +56,11 @@ public function create_coupon( $entry, $form ) {
 			$coupon_name = ($coupon_name == ''? $coupon_code : $coupon_name);
 		}
 
+		$limit_value = rgar($entry, $this->_args['meta']['coupon_limit']);
```

Comment:
If the user wants to actually set a static limit there is a chance that it will match a field ID and be incorrectly replace. My instinct here is to allow this property to be a [callable function](https://www.php.net/manual/en/function.call-user-func.php) to which we can pass the $entry.

If you're not sure what I mean, let's implement it together! 🙂